packages/cli/openapi-ir-to-fern/src/converters/convertToServices.ts

```diff
@@ -47,10 +50,11 @@ export function convertToServices({
         const service = services[file];
         if (service != null) {
             if (endpointId in service.value.endpoints) {
-                throw new Error(
-                    `The OpenAPI Spec has conflicting sdk method names. See ${endpoint.method.toUpperCase()} ${
+                const firstEnd = endpoints.find((e) => e.sdkName?.methodName === endpointId);
+                context.logger.error(
+                    `The OpenAPI Spec has conflicting SDK method names. See ${endpoint.method.toUpperCase()} ${
```

Comment:
May be more helpful to log something like: 

```
Multiple endpoints have conflicting names. See below: 
  POST /users has the name `getUser` (line 123)
  PUT /users has the name `getUser` (line 123)
You can change the names using operationId or the extension `x-fern-sdk-method-name`. 
```

Reply:
Gotcha, yeah I update this to group all of the conflicts together with more details

Reply:
Sounds good -- my guess is that line number is too hard and can follow up with that later

Reply:
And then lets add a test to test this case in the `ete-tests` or the `openapi-ir-to-fern` tests

Reply:
I updated the conflicts to format like below, and added a test to `ete-tests`
```
Multiple endpoints have conflicting names for 'hello':
- GET /hello
- GET /hellov2
Multiple endpoints have conflicting names for 'goodbye':
- GET /goodbye
- GET /goodbyev2
You can change the names using operationId or the extension `x-fern-sdk-method-name`.
```